### PR TITLE
Feature/#1 support helm resource

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #1

### CODE REVIEW

File: resources/helm.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        EOF
+    only_if 'which helm'
+  end
+
+  download_url = "https://storage.googleapis.com/kubernetes-helm/helm-#{version}-#{platform}-#{arch}.tar.gz"
+
+  bash 'install helm' do
+    code <<-EOH
+      curl #{download_url} | tar -xvz
+      mv linux-amd64/helm /usr/local/bin/helm
```
Comment (hoatle) [inline resources/helm.rb]: linux-amd64 is hard-coded here?

File: resources/helm.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      EOH
+  end
+end
+
+action :remove do
+  execute 'remove helm' do
+    command "rm -rf #{binary_path}"
```
Comment (hoatle) [inline resources/helm.rb:70]: possible bug, however, we'll fix it by writing tests for these soon.

File: spec/unit/resources/helm_install-default_spec.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#
```
Comment (hoatle) [inline spec/unit/resources/helm_install-default_spec.rb]: file naming, helm_install_default_spec.rb instead

File: spec/unit/resources/helm_install-default_spec.rb [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+require 'spec_helper'
+
+describe 'kubernetes-stack-test::helm_install_default' do
+  context 'When all attributes are default, on ubuntu 16.04' do
+
+    let(:chef_run) { ChefSpec::SoloRunner.new(step_into: 'helm', platform: 'ubuntu', version: '16.04').converge(described_recipe) }
+
+    before do
+      stub_command('which helm').and_return(true)
```
Comment (hoatle) [inline spec/unit/resources/helm_install-default_spec.rb]: and_return('/usr/local/bin/helm')

**Review by hoatle — CHANGES_REQUESTED** [review]
@hieptranquoc review done, please check my comments.

**Comment by hieptranquoc** [comment]
@hoatle review my PR again please

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**